config/settings.yml

```diff
@@ -4,6 +4,7 @@ HOSTNAME: <%= (`echo $HOSTNAME`).gsub('.stanford.edu', '') %>
 STACKS_URL: "https://stacks.stanford.edu/image"
 REQUESTS_URL: "https://host.example.com/requests/new"
 SSRC_REQUESTS_URL: "http://host.example.com/link.ssds_request_form"
+HOOVER_REQUESTS_URL: "https://host.example.com/aeon.dll?Action=10&Form=20"
```

Comment:
I'm wondering if we need to obscure this URL at all?  It'll be publically available in the SearchWorks UI (and was in the ticket).

Anybody have thoughts on just checking in the url into this settings file?